022-feature-gates.md

```diff
@@ -0,0 +1,125 @@
+# Feature Gates
+
+This proposal suggests to add support for Feature Gates in the Strimzi operators.
+Feature Gates should give us additional options how to control and mature different behaviors in the operators.
+
+## Current situation
+
+Currently, any features or behaviors of the operator are controlled by one of these two mechanisms:
+
+* The Strimzi custom resources (as a form of an API)
+* The state of the Kubernetes resources
```

Comment:
```suggestion
* The state of the Kubernetes resources
* Environmental variables in the operator deployment
```
I think it's worth calling this out explicitly as if differs slightly from the other kube resources